packages/product-react/src/components/List/List.scss

```diff
@@ -0,0 +1,51 @@
+@import url('https://www-static.wework.com/apercu/apercu.css');
+@import url('https://www-static.wework.com/apercu/apercu_mono.css');
+
+$list-font-family: 'Apercu';
```

Comment:
Do we need to set the font-family here? I was thinking that it would inherit it from somewhere else. But, if we have to set it, we should probably have some fallback fonts. 

I just grabbed this from wework.com haha:
`'Apercu-Regular', 'Apercu', Avenir, -apple-system, BlinkMacSystemFont, 'Helvetica Neue', Helvetica, 'Calibri', 'Roboto', Arial, sans-serif`

Reply:
ok done